com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -1180,13 +1198,17 @@ void RenderStencilLights(ScriptableRenderContext context, CommandBuffer cmd, ref
             if (m_FullscreenMesh == null)
                 m_FullscreenMesh = CreateFullscreenMesh();
 
-            using (new ProfilingScope(cmd, m_ProfilingSamplerDeferredStencilPass))
+            // TODO: verify if this is really required as _CLEARCOAT is already set in the material if it's enabled
+            using( new KeywordScope(cmd, ShaderKeywordStrings._CLEARCOAT) )
```

Comment:
Should we set _CLEARCOAT as a global keyword? The material does set it if enabled here https://github.com/Unity-Technologies/Graphics/blob/febc85c20f7eb6876d062896bf6a719817c89357/com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs#L284